internal/campaigns/debug.go

```diff
@@ -0,0 +1,7 @@
+package campaigns
+
+import "github.com/sourcegraph/src-cli/internal/output"
+
+// DebugOut can be used to print debug messages in development to the TUI.
+// For that it needs to be set to an actual *output.Output.
+var DebugOut output.Writer = output.NoopWriter{}
```

Comment:
I need this and I'll fight for this to stay in 😛

Why do I need this? Because when I want to debug something my `fmt.Print` output gets eaten by our TUI. With this I can set `campaigns.DebugOut = out` and then `campaigns.DebugOut.Writef("foobar")`. Doesn't always work, but this is a good first step.